object.js

```diff
@@ -259,7 +259,7 @@ define(function (require) {
 
 	// for better compression
 	function hasProp (object, name) {
-		return object.hasOwnProperty(name);
+		return Object.hasOwnProperty.call(object, name);
```

Comment:
This is a bit confusing, imho.  The way this is written, it appears that there is a method, `hasOwnProperty` defined on the `Object` constructor.  However, `hasOwnProperty` is only defined on `Function.prototype`, not on `Object`.  Perhaps you could use the `refProto` variable from [above](https://github.com/cujojs/poly/blob/master/object.js#L73)?

`return refProto.hasOwnProperty.call(object, name);`